qemu/tests/blockdev_inc_backup_merge_with_nonexist_bitmap.py

```diff
@@ -0,0 +1,66 @@
+from virttest.qemu_monitor import QMPCmdError
+
+from provider.block_dirty_bitmap import block_dirty_bitmap_add
+from provider.blockdev_live_backup_base import BlockdevLiveBackupBaseTest
+
+
+class BlkdevIncMergeWithNonexistBitmap(BlockdevLiveBackupBaseTest):
+    """Merge bitmaps with a non-exist bitmap to the target bitmap"""
+
+    def __init__(self, test, params, env):
+        super(BlkdevIncMergeWithNonexistBitmap, self).__init__(
+            test, params, env)
+        self._merged_bitmaps = params.objects('bitmap_merge_list')
+        self._merged_target = params['bitmap_merge_target']
+
+    def add_one_bitmap(self):
+        args = {'target_device': self._source_nodes[0],
+                'bitmap_name': self._merged_bitmaps[0]}
+        block_dirty_bitmap_add(self.main_vm, args)
+
+    def merge_two_bitmaps(self):
+        target_bitmap = {'node': self._source_nodes[0],
+                         'name': self._merged_target,
+                         'disabled': True}
+        merged_bitmap = {'node': self._source_nodes[0],
+                         'bitmaps': self._merged_bitmaps,
+                         'target': self._merged_target}
+        job_list = [
+            {'type': 'block-dirty-bitmap-add', 'data': target_bitmap},
+            {'type': 'block-dirty-bitmap-merge', 'data': merged_bitmap}
+        ]
+        try:
+            self.main_vm.monitor.transaction(job_list)
+        except QMPCmdError as e:
+            nonexist_bitmap = self._merged_bitmaps[1]
+            qmp_error_msg = self.params.get("qmp_error_msg") % nonexist_bitmap
+            if qmp_error_msg not in str(e.data):
+                self.test.fail(str(e))
+        else:
+            self.test.fail("Can merge with a non-exist bitmap:%s"
+                           % nonexist_bitmap)
```

Comment:
nonexist_bitmap -> self._merged_bitmaps[1]
If the program enters into the else block, nonexist_bitmap is a var undefined

Reply:
have updated, thanks zhenchao.